### DISCUSSION TITLE
Heading's background color is missing

### DISCUSSION THREAD

**Discussion by guiribmedeiros** [laravel/nova-issues#5018] [Ideas & Feature Requests] [answered]
- Laravel Version: 9.37
- Nova Version: 4.18
- PHP Version: 8.0.24
- Operating System and Version: macOS Monterey 12.6.1
- Browser type and version:  Google Chrome 106.0.5249.119

### Description:
- Heading is not displaying its background color
- It's pretty hard to distinguish from other fields

### Detailed steps to reproduce the issue on a fresh Nova installation:
- Add the Heading field
- `Heading::make('Webhook')`

![Screen Shot 2022-11-03 at 21 21 26](https://user-images.githubusercontent.com/15630096/199859120-43167644-d913-4483-8443-77ff1dcbfd8b.png)

**Comment by crynobone** [comment, marked as answer]
The changes to remove the background color were intentionally made so I would need to mark this as a change request.

**Comment by ftadra** [comment]
The color change seems to have change the UX of the page due to the reduced contrast it creates between elements and the heading. If it was an intentional change, I'd argue is not a very user friendly one. Any way to revert this?

I'd also suggest adding classes to common elements so we can overwrite some of these styles if changes like that are to become a trend.
